apps/cookbook/src/app/examples/menu-example/examples/advanced.ts

```diff
@@ -3,10 +3,21 @@ import { Component } from '@angular/core';
 const config = {
   selector: 'cookbook-menu-advanced-example',
   template: `<kirby-menu [closeOnSelect]="false">
-  <kirby-item (click)="actionClicked()">
+  <kirby-item selectable="true">
```

Comment:
Consider using property binding (resolves to boolean) instead of plain string:
```suggestion
  <kirby-item [selectable]="true">
```